.github/workflows/aws_compute_ec2_single_region_tests.yml

```diff
@@ -83,121 +128,249 @@ jobs:
-                  echo "TF_PREFIX=${prefix}" >> "$GITHUB_ENV"
+            - name: Install asdf tools with cache for the project
+              uses: camunda/infraex-common-config/./.github/actions/asdf-install-tooling@6493c4661ae26f7c81689994c4c75e8f42b67f6a # 1.4.5
 
-            - name: Replace matrix dots with hyphen
+            - name: Install zbctl
               run: |
                   set -euo pipefail
+                  npm i -g zbctl
+                  zbctl version
+            - name: Configure AWS CLI
+              uses: ./.github/actions/aws-configure-cli
+              with:
+                  vault-addr: ${{ secrets.VAULT_ADDR }}
+                  vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+                  vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+                  aws-profile: ${{ env.AWS_PROFILE }}
+                  aws-region: ${{ env.AWS_REGION }}
 
-                  version=$(echo "${{ matrix.camunda }}" | tr '.' '-')
-                  echo "CAMUNDA_HYPHENS=$version" >> "$GITHUB_ENV"
-
-            # yamllint disable rule:line-length
-            - name: Configure Terraform Backend
+            - name: 📁 Get a copy of the reference architecture
+              timeout-minutes: 10
               run: |
+                  # run it as specified in the doc
                   set -euo pipefail
 
-                  cp ${{ github.workspace }}/aws/compute/ec2-single-region/test/fixtures/*.tf ${{ github.workspace }}/aws/compute/ec2-single-region/terraform/
-                  echo "TF_CLI_ARGS_init=-backend-config='bucket=${{ env.S3_BACKEND_BUCKET }}' -backend-config='key=state/${{ env.TF_PREFIX }}/terraform.tfstate' -backend-config='region=${{ env.S3_BUCKET_REGION }}' -backend-config='encrypt=true'" >> "$GITHUB_ENV"
-            # yamllint enable rule:line-length
-            - name: Set Camunda version
-              working-directory: aws/compute/ec2-single-region/scripts
+                  ./aws/compute/${{ matrix.scenario.name }}/procedure/get-your-copy.sh
+                  tree
+
+            - name: Set current Camunda version
+              id: camunda-version
               run: |
                   set -euo pipefail
-
-                  # We need to replace the string without substitution except for that of GitHub Actions
-                  # shellcheck disable=SC2016
-                  sed -i '0,/CAMUNDA_VERSION/s/CAMUNDA_VERSION=.*/CAMUNDA_VERSION=${CAMUNDA_VERSION:-"${{ matrix.camunda }}"}/' camunda-install.sh
-            # potential to rework to dynamic matrix instead or local file that contains version dict
-            - name: Overwrite previous version
+                  # TODO: Intended override CAMUNDA_VERSION=$(cat .camunda-version)
+                  CAMUNDA_VERSION=8.6
+                  echo "CAMUNDA_VERSION=$CAMUNDA_VERSION" | tee -a "$GITHUB_OUTPUT"
+            - name: Get latest full Camunda version
+              id: camunda-version-full
```

Comment:
was just an idea to use the `camunda-version` file to then based on that resolve the current latest "stable" version from Artifactory.

But maybe just using Renovate and have it hardcoded at the top may be easier and better. Also aligns with the other workflows we have.
Mh, the more I think about it, the more I'd probably go for the way we usually do it.